src/App.tsx

```diff
@@ -34,6 +38,26 @@ function App() {
     {
       path: "Dashboard",
       element: <Dashboard />,
+      children: [
+        {
+          path: "Home",
+          element: <Home />,
+        },
+        {
+          path: "Feed",
+          element: <Feed />,
+        },
+        {
+          path: "Messages",
+          element: <Messages />,
+          children: [
+            {
+              path: "ChatBox",
```

Comment:
You do not need to add another route for this chatbox just add it tto the messages component directly

Reply:
okay